collections/_posts/2022-06-21-3D+tCmrQuantification.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: review
+title: "Automatic 3D+t four-chamber CMR quantification of the UK biobank: integrating imaging and non-imaging data priors at scale"
+tags: deep-learning MRI heart multi-modal
+author: "Nathan Painchaud"
+cite:
+    authors: "Yan Xia, Xiang Chen, Nishant Ravikumar, Christopher Kelly, Rahman Attar, Nay Aung, Stefan Neubauer, Steffen E Petersen, Alejandro F Frangi"
+    title:   "Automatic 3D+t four-chamber CMR quantification of the UK biobank: integrating imaging and non-imaging data priors at scale"
+    venue:   "Medical Image Analysis (2022)"
+pdf: "https://doi.org/10.1016/j.media.2022.102498"
+---
+
+
+# Highlights
+- New *Multi-Cue Shape Inference Network* (MCSI-Net) that uses multi-view image data as well as non-imaging demographic
+data to predict 3D cardiac shapes of all four cardiac chambers;
+- Experiments to show that integrating multi-view and multi-modal data allows the network to accurately reconstruct 3D
+cardiac shapes with less image data (1/2 to only 1/4 of the slices), which could speed-up CMR scan acquisitions.
```

Comment:
May be specifying the full name of CMR (Cardiovascular Magnetic Resonance) for once. It took me some time to figure out what CMR is. ;)